src/migrations/versions/1dfc5a2d30e7_add_saas_config_to_connection_config.py

```diff
@@ -0,0 +1,43 @@
+"""add saas config to connection config
+
+Revision ID: 1dfc5a2d30e7
+Revises: 07014ff34eb2
+Create Date: 2022-02-09 23:27:24.742938
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = "1dfc5a2d30e7"
+down_revision = "07014ff34eb2"
```

Comment:
just a note that since bigquery has been merged, you'll need to bump this revision id

Reply:
💯

Reply:
Fixed!